src/osd/osd_types.h

```diff
@@ -3986,7 +3986,7 @@ struct object_info_t {
 
   flag_t flags;
 
-  static string get_flag_string(flag_t flags) {
```

Comment:
-1 also, it simply doesn't make sense. Static version, as @liewegas wrote, can be used to translate flag_t to string without constructing an OSDMap instance (that could be desireable, for example, for performance reasons). Your change forces developers to construct an OSDMap instance each time they want to use this function.